src/content/ru/2021/seo.md

```diff
@@ -0,0 +1,758 @@
+   sheets_gid="718210755",
+   sql_file="markup-stats.sql"
+   )
+}}
+
+Однако в 2021 году темпы распространения AMP продолжали расти. Атрибут AMP был обнаружен на 0.09% страниц для компьютеров и на 0.22% страниц для мобильных страниц. Это больше, чем 0.06% на компьютерных и 0.15% на мобильных страницах в 2020 году.
+
+## Интернационализация {internationalization}
+
+<figure>
+  <blockquote>Если у вас несколько версий страницы на разных языках или для разных регионов, помогите Google идентифицировать их. Тогда в результатах поиска будут показываться те языковые версии ваших страниц, которые лучше всего подходят для пользователя.</blockquote>
+  <figcaption>— <cite><a hreflang="en" href="https://developers.google.com/search/docs/advanced/crawling/localized-versions">SEO-документация Google</a></cite></figcaption>
+</figure>
+
+Чтобы поисковые системы знали о локализованных версиях ваших страниц, используйте теги `hreflang`. Атрибуты `hreflang` также используются <a hreflang="ru" href="https://yandex.com/support/webmaster/yandex-indexing/locale-pages.html?lang=ru">Яндексом</a> и Bing ([в некоторой степени](https://twitter.com/facan/status/1304120691172601856)).
+
+{{ figure_markup(
+   image="hreflang-usage.png",
+   caption="Гистограмма главных атрибутов hreflang.",
+   description="Горизонтальная гистограмма показывает использование атрибута hreflang. Самым популярным значением атрибута hreflang было `en` (английская версия), и атрибуты hreflang (на всех языках) использовались менее чем на 5% страниц для компьютеров и мобильных устройств.",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1149395895&format=interactive",
+   sheets_gid="866829014",
+   sql_file="hreflang-link-tag-usage.sql",
+   width=600,
+   height=546
+   )
+}}
+
+Атрибут hreflang использовался на 9.0% страниц для компьютеров, и на 8.4% мобильных страниц.
+
+Существует три способа реализации `hreflang`: в HTML-элементах `<head>`, в HTTP-заголовках `Link` и с помощью карт сайта XML. Наши данные не включают в себя данные из карт сайтов XML. 
+
+Самый популярный атрибут hreflang — `"en"` (английская версия). Его используют 4.75% главных страниц для мобильных устройств, и 5.32% — для компьютеров.
+
+Значение `x-default` (также называемое резервной версией) используется в 2.56% случаев на мобильных устройствах. Другими популярными языками, используемыми в атрибутах `hreflang`, являются французский и испанский.
+
+Для Bing [`hreflang` является "гораздо более слабым сигналом", чем HTTP-заголовок `content-language`](https://twitter.com/facan/status/1304120691172601856).
+
+Как и многие другие параметры SEO, у [`content-language`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Language) есть несколько методов реализации, включая:
```

Comment:
This MDN article is available in Russian to let's link there:

```suggestion
Как и многие другие параметры SEO, у [`content-language`](https://developer.mozilla.org/ru/docs/Web/HTTP/Headers/Content-Language) есть несколько методов реализации, включая:
```

Reply:
Sorry, fixed it.